packages/wallet/test/hardware/trezor/TrezorKeyAgent.test.ts

```diff
@@ -1,19 +1,27 @@
 import * as Crypto from '@cardano-sdk/crypto';
-import { AddressType, CommunicationType, SerializableTrezorKeyAgentData, util } from '@cardano-sdk/key-management';
+import {
+  AddressType,
+  Bip32Account,
+  CommunicationType,
+  SerializableTrezorKeyAgentData,
+  util
+} from '@cardano-sdk/key-management';
 import { AssetId, createStubStakePoolProvider, mockProviders as mocks } from '@cardano-sdk/util-dev';
 import { Cardano, Serialization } from '@cardano-sdk/core';
 import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
 import { HexBlob } from '@cardano-sdk/util';
 import { InitializeTxProps, InitializeTxResult } from '@cardano-sdk/tx-construction';
-import { PersonalWallet, setupWallet } from '../../../src';
+import { PersonalWallet } from '../../../src';
 import { TrezorKeyAgent } from '@cardano-sdk/hardware-trezor';
+import { firstValueFrom } from 'rxjs';
 import { dummyLogger as logger } from 'ts-log';
 import { mockKeyAgentDependencies } from '../../../../key-management/test/mocks';
 
 describe('TrezorKeyAgent', () => {
   let wallet: PersonalWallet;
   let keyAgent: TrezorKeyAgent;
```

Comment:
Since `keyAgent` is declared as`trezorKeyAgent` [here](https://github.com/input-output-hk/cardano-js-sdk/blob/7e67e9da9641a1ea43431acb0591a23c994fab65/packages/wallet/test/hardware/trezor/TrezorKeyAgent.test.ts#L36) we should rename it and use in tests.
That is the reason why the tests are failing. 

![Screenshot 2023-12-06 at 11 46 22](https://github.com/input-output-hk/cardano-js-sdk/assets/18653950/729d5d38-9d5f-4cd0-901c-880b04670f5a)


p.s. I will check Ledger tests as well.

Reply:
Nice catch, it was passing before I rebased on master this morning. 6d254d4dd560b1bca6130093faad58d6441a617a